### ISSUE TITLE
Next.js should ignore all Moment locales via opt-in

### ISSUE THREAD

**Issue by Timer** [zeit/next.js#8300]
Next.js should ignore all moment locales and request the user opt-into the feature, until it becomes a default at some point in the future.

```js
      // Moment.js is an extremely popular library that bundles large locale files
      // by default due to how Webpack interprets its code. This is a practical
      // solution that requires the user to opt into importing specific locales.
      // https://github.com/jmblog/how-to-optimize-momentjs-with-webpack
      new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
```

---

Maybe it's worth tuning Lodash with `babel-plugin-transform-imports` or something similar?

**Comment by kachkaev** [comment]
This looks like a breaking change 🤔

**Comment by Timer** [comment]
@kachkaev that's why the behavior is opt-in @kachkaev!

**Comment by kachkaev** [comment]
I thought the ignore plugin would be just injected and that's it. If it's behind a option, no Next.js projects will be harmed :)

**Comment by timneutkens** [comment]
This was landed in #8345

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
